setup.cfg

```diff
@@ -64,6 +64,9 @@ testing =
     pytest-pylint == 0.18.0
     jsons==1.4.2
     mockito==1.2.2
+cicd =
+    twine == 3.3.0
+    wheel == 0.36.2
```

Comment:
Isn't `wheel` included in the Python image? Is there some reason to pin it?

Reply:
The virtual environment used for the action is this one: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md which afaik does not contain `wheel`. No reason to pin it to that specific version though, should be fixed in my newest commit.